go/teams/handler.go

```diff
@@ -135,16 +140,19 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 	case keybase1.TeamInviteCategory_EMAIL:
 		// nothing to verify, need to trust the server
 	case keybase1.TeamInviteCategory_KEYBASE:
-		uid, err := keybase1.UIDFromString(string(invite.Name))
+		// XXX and check the seqno
```

Comment:
what's happening with this TODO?  Seems like a big security concern.

Reply:
Thanks for the reminder, yes, I'll add it...it wasn't being checked before because it was just looking at UID.